src/muso_cdi_test/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="app_name">Muso CDI Test</string>
+  <string name="app_host">muso-cdi.dev.medicmobile.org</string>
```

Comment:
Great job here @adrienlemec.
Are we planning to create production's ( `muso-cdi.app.medicmobile.org` ) flavor at another stage?

Reply:
Oh, I missed that it was pointing to the dev URL. I don't think it's necessary to create an official APK with this method pointing to the dev instance. We can easily create it locally and share it via email, drive etc. Also, the number of flavors is already too high, let's not make it even more crowded.

Thanks for catching it, Marina.

Reply:
Alright, I have the APK locally. Let me share it with everyone.

Reply:
Do we still want to merge it or the APK created locally would suffice?